Dev/Git/get_pull_requests_bitbar.1h.rb

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-
+# frozen_string_literal: true
 
 # <bitbar.title>PR Counts for Github and Bitbucket</bitbar.title>
 # <bitbar.version>v1.0</bitbar.version>
```

Comment:
@MarcoCabazal  please increment the version to indicate this is a breaking change.

Reply:
Thanks, will do just that.